service/policy/kasregistry/key_access_server_registry.proto

```diff
@@ -158,13 +169,14 @@ message ListPublicKeysResponse {
 }
 
 message ListPublicKeyMappingRequest {
-  // Optional
-  string kas_id = 1;
-  // Future filter by fields
-  // // Optional
-  // string kas_name = 2;
-  // // Optional
-  // string kas_uri = 3;
+  oneof identifier {
```

Comment:
I am not concerned about these breaking changes as we haven't released this new functionality yet.